### ISSUE TITLE
It's difficult to see which parts are tasks compared to information.

### ISSUE THREAD

**Issue by azizi-a** [oliverjam/learn-express-middleware#1]
It would be helpful if the parts the student is expected do were highlighted in some way. Perhaps block quotes could be used for tasks.

**Comment by oliverjam** [comment]
You're right, atm you have to read everything to see where the tasks are.

Would the same thing as previous workshops be okay here? A **Challenge** heading with a numbered list of tasks to complete, followed by the solution?

**Comment by azizi-a** [comment]
That would work well, yes. It still might be nice to have something to highlight them further for scanning through when comparing answers. Perhaps some underlining on the word **Challenge** its self?

**Comment by oliverjam** [comment]
Unfortunately we can't change GitHub's CSS. I'll see what I can do with standard markdown

**Comment by azizi-a** [comment]
Hadn't resiled you couldn't underline in markdown yet. Ha. No worries.

**Comment by azizi-a** [comment]
How about a _*_ next to each __Challenge__, as in __Challange *__? That'd work excellently for me at least.
